scenarios/action_recognition/02_training_hmdb.ipynb

```diff
@@ -34,7 +34,7 @@
   },
```

Comment:
<p>Good notebook.</p><p>Could point out here (to make it even more explicit) that the paper reports an accuracy of XX% on the popular research benchmark dataset HMDB-51, and with this notebook we achieve XX%, showing that our implementation is indeed correct.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/microsoft/computervision-recipes/pull/535/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>